src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/RazorLanguageServer.cs

```diff
@@ -102,6 +103,9 @@ protected override ILspServices ConstructLspServices()
         services.AddHoverServices();
         services.AddTextDocumentServices();
 
+        // Signature Help
+        services.AddRegisteringHandler<SignatureHelpEndpoint>();
```

Comment:
Should this move into the static local method `static void AddHandlers(IServiceCollection services)` to be with the other `AddRegisteringHandler` calls ?

Reply:
Yeah, good point. Missed that.

I hate DI without discovery 😛